src/PowerTools/source.extension.vsixmanifest

```diff
@@ -14,35 +13,23 @@ When right-clicking on a file containing a derived DbContext class, the followin
 1) View Entity Data Model XML - Displays the EDMX XML representing the underlying Code First model.
```

Comment:
Looking at this page, maybe the short description should just be changed to:
"Preview of useful design-time features for DbContext."

(The description below on that page is entered manually, and is independent of the contents on the VSIX package)